client/testutil/suite.go

```diff
@@ -101,12 +101,12 @@ func (s *TxConfigTestSuite) TestTxBuilderSetSignatures() {
 
 	// set SignatureV2 without actual signature bytes
 	sigData1 := &signingtypes.SingleSignatureData{SignMode: signModeHandler.DefaultMode()}
-	sig1 := signingtypes.SignatureV2{PubKey: pubkey, Data: sigData1}
```

Comment:
These 2 and 4 seem totally arbitrary in the test case.

I saw no explicit sequence number in the previous code. If this is pulled from the account in some other function, maybe you can do `acct.Sequence` instead of 2. If they are arbitrary (but need to be the same with the same 2 below, then can you assign them to a variable? (Magic undeclared constants make code hard to maintain and test code hard to follow. It can be as simple as:

```go
seq1 := 2 
sig1 := signingtypes.SignatureV2{PubKey: pubkey, Data: sigData1, Sequence: seq1}
```

Unless that 2 is correllated with some other configuration, where it should have a reference to show the connection